deployment/network-operator/templates/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -84,6 +84,10 @@ spec:
             "resourceName": {{ $element.name | quote }},
             "selectors": {
               "vendors": {{ $element.vendors | default list | toJson }},
+              "devices": {{ $element.devices | default list | toJson }},
+              "drivers": {{ $element.drivers | default list | toJson }},
+              "ifNames": {{ $element.ifNames | default list | toJson }},
```

Comment:
sriov network device plugin does not have this selector:  https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin#extended-selectors-for-device-type-netdevice

Reply:
https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin#device-selectors

Reply:
we should align to have the following selectors.

vendors, devices, drivers, pfNames, pciAddresses, rootDevices, linkTypes